src/stavenote.ts

```diff
@@ -692,7 +641,8 @@ export class StaveNote extends StemmableNote {
    * @returns true if this note is a type of rest. Rests don't have pitches, but take up space in the score.
    */
   isRest(): boolean {
-    return this.glyphProps.rest;
+    const val = this.glyphProps.codeHead;
+    return val >= '\ue4e0' && val <= '\ue4ff';
```

Comment:
@rvilarl Does this work? What type is codeHead? Is it a string? Can we do inequality checks with the strings here?

I think it is safer to do this type of check with numeric values.

Reply:
From my most recent review, I think this one is the most confusing. We shouldn't really be doing mathematical comparisons against string values.